README.md

```diff
@@ -32,8 +26,11 @@ $ which dhis2-subscriptions
 * Add crontab entries (`crontab -e`) to send notifications and newsletters to subscribers. An example:
 
 ```
-*/5 * * * *   chronic /path/to/bin/dhis2-subscriptions --config-file=/path/to/your/config.json send-notifications
-00  8 * * MON chronic /path/to/bin/dhis2-subscriptions --config-file=/path/to/your/config.json send-newsletters
+*/5 * * * *   chronic /path/to/bin/dhis2-subscriptions generate-events --config-file=/path/to/your/config.json 
```

Comment:
I have moved all the options to the commands because I found it confusing to have --config-file generic and --ge by command. There is no problem in setting the config-file option as generic again. @adrianq?

Reply:
Don't worry, it's ok as it is.